Fundamentals/Backoffice/Property-Editors/Built-in-Property-Editors/Block-Editor/Block-Grid-Editor/index.md

```diff
@@ -160,15 +161,22 @@ To scale an Area, click and drag the scale-button in the bottom-right corner of
 
 ### Area configuration
 
+An area 
+
 ![Block Grid - Area Configuration](images/BlockGridEditor_AreasConfiguration.png)
 
-* **Alias** - The alias is printed by `GetBlockGridHTML()`. Use the alias to target the Element representing the area.
+* **Alias** - The alias is used to identify this Area.
+It begin printed by `GetBlockGridHTML()` And used as name for the Area slot in Custom Views.
+The alias is also available for CSS Selectors to target the HTML-Element representing a Area.
 
 * **Create Button Label** - Overwrites the Create Button Label of the Area.
 
 * **Number of blocks** - Determines the total number of Blocks in an Area.
 
-* **Allowed block types** - Defines the types of Blocks or Groups of Blocks that are allowed in the Area. Optionally, you can also set how many Blocks of each type/group should be present. The Blocks can be limited for each specific Block Type in an Area.
+* **Allowed block types** - When this is empty, all Blocks with Permissions for creation in Areas, will be available. This can be overwritten by specifying the allowed Blocks.
+Define the types of Blocks or Groups of Blocks that are allowed. Additionally, you can also set how many Blocks of each type/group should be present.
```

Comment:
Should this line be part of the "Allowed block types" list item in the line above?

Reply:
Yep, thats right:-)